Update develop

ndasilver opened the pull request:
#### What does this PR do?
this PR update all the changes merged from gh-pages

ndasilver committed:
added link for my gh-pages in readme file

ndasilver committed:
feat(Dashborard): Created Dashbord
- created UI dashboard for the user after signin
- this contains an option of creating a new account
- this page contains an option for viewing user account transactions

ndasilver committed:
chore:
- changed some syles and url linking

ndasilver committed:
Merge pull request #4 from ndasilver/ft-ui-dashboard-page-171147903

#171147903: Created Dashbord

ndasilver committed:
feat(create_account): create new account page
- created ui for creating new account while a user wants to create one
- here a user will see a button for creating a new account in the dashboard
[Finishes #171091650]

ndasilver committed:
Merge pull request #5 from ndasilver/ft-ui-user-create-account-page-171091650

#171091650: create new account page

ndasilver committed:
feat(account transaction history): created account transaction page
- Created account transction ui for the user
[Finishes #171091672]

ndasilver committed:
feat(account transaction history): created account transaction page
- Created account transction ui for the user
[Finishes #171091672]

ndasilver committed:
Merge pull request #6 from ndasilver/ft-create-account-transaction-history-page-171091672

#171091672: Ft create account transaction history page

ndasilver committed:
chore: changes a littel style

ndasilver committed:
chore: changes a littel style

ndasilver committed:
Merge pull request #8 from ndasilver/ft-create-account-transaction-history-page-171091672

Ft little styles

ndasilver committed:
feat(ui for viewing all accounts:
- I created ui for viewing all bank accounts as an admin/staff
- This contains some options; for viewing a specific account and deleting a specific account
[Finish #171172313]

fridolinho commented on UI/css/styles.css:
```diff
@@ -217,7 +221,7 @@ header ul li span.user-name {
     padding-left: 40px;
```
Try Use the padding shorthand property with four values

ndasilver replied:
Thank you. I will work on them

fridolinho reviewed (commented):
Great keep pushing.

ndasilver committed:
feat(created view a specific bank account record page):
- From the bank accounts list an admin/staff can click on view with a selected bank account and view all informations about that particular account
- This page contains all about the User account information, like: his/her name, bank account number, status, balance and the date when the account was created.
[Finish #171091672]

ndasilver committed:
Merge pull request #9 from ndasilver/ft-create-account-page-171172313

#171172313: Ft create account page

ndasilver committed:
feat(users): created users page feature
- created ui for the users
- created activate and deactivate option in the user page ui
[Finish #171179716]

ndasilver committed:
feat(users): created users page feature
- Added an option for creating new user
[Finish #171179716]

ndasilver committed:
Merge pull request #10 from ndasilver/ft-create-user-page-171179716

#171179716: created users page feature
